README.md

```diff
@@ -101,14 +101,409 @@ download or scan any previously scanned sources, the Scanner can be configured (
+* SBT
+* Subversion
+* Yarn
+
+**2. Download & Install ORT**
+
+In future we will provide binaries of the ORT tools, but currently you have to build the tools on your own. First
+download the source code from GitHub (replace `[ort-path]` with the path you want to clone ORT to):
+
+```bash
+git clone https://github.com/heremaps/oss-review-toolkit.git [ort-path]
+```
+
+To build the tools run:
+
+```bash
+cd [ort-path]
+./gradlew installDist
+```
+
+This will create binaries of the tools in their builds folders, for example the analyzer binary can be found in
+`analyzer/build/install/analyzer/bin/analyzer`. To get the command line help for tool run it with the `--help` option:
+
+```bash
+$ analyzer/build/install/analyzer/bin/analyzer --help
+Usage: analyzer [options]
+  Options:
+  * --input-dir, -i
+      The project directory to scan.
+  * --output-dir, -o
+      The directory to write dependency information to.
+    --package-curations-file
+      A YAML file that contains package curation data.
+    --ignore-versions
+      Ignore versions of required tools. NOTE: This may lead to erroneous 
+      results. 
+      Default: false
+    --merge-results
+      Merge all results into a single results file. The individual scan 
+      results files for each build file will still be created.
+      Default: false
+    --allow-dynamic-versions
+      Allow dynamic versions of dependencies. This can result in unstable 
+      results when dependencies use version ranges. This option only affects 
+      package managers that support lock files, like NPM.
+      Default: false
+    --package-managers, -m
+      A list of package managers to activate.
+      Default: [Gradle, Maven, SBT, NPM, PIP]
+    --output-format, -f
+      The data format used for dependency information.
+      Default: YAML
+      Possible Values: [JSON, YAML]
+    --debug
+      Enable debug logging and keep any temporary files.
+      Default: false
+    --stacktrace
+      Print out the stacktrace for all exceptions.
+      Default: false
+    --info
+      Enable info logging.
+      Default: false
+    --help, -h
+      Display the command line help.
+```
+
+**3. Download the Vue.js source code**
+
+Before scanning `mime-types` its source code has to be downloaded. For reliable results we use version 2.1.18 (replace
+`[mime-types-path]` with the path you want to clone `mime-types` to):
+
+```bash
+git clone https://github.com/jshttp/mime-types.git [mime-types-path]
+cd [mime-types-path]
+git checkout 2.1.18
+```
+
+**4. Run the analyzer on `mime-types`**
+
+The next step is to run the `analyzer`. It will create a JSON or YAML output file containing the full dependency tree of
+`mime-types` including the meta-data of `mime-types` and its dependencies.
+
+```bash
+# The easiest way to run the analyzer. Be aware that the [output-path] directory must not exist.
+[ort-path]/analyzer/build/install/analyzer/bin/analyzer -i [mime-types-path] -o [output-path]
```

Comment:
If we omit the path to clone to, I'd simply start the path with `analyzer/...` here (like you also do in the shell snippets).